corehq/apps/style/templates/style/includes/analytics_google.html

```diff
@@ -47,11 +47,25 @@
                     // This page isn't being redirected
                     trackFunction();
                 } else {
+                    // Track how many trackLinkHelper-related handlers
+                    // this event has, so we only actually click
+                    // once, after they're all complete.
+                    if (!event.data) {
+                        event.data = {};
```

Comment:
super minor: this could be `event.data = event.data || {}`
same with below. non-blocking just slightly more concise